packages/dataviews/README.md

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
I left comments by the real changes, the rest are formatting done by `prettier`.